### DISCUSSION TITLE
How to conform to the HRecord Protocol?

### DISCUSSION THREAD

**Discussion by Narsail** [aaronpearce/Harmony#14] [Q&A] [answered]
Hi @aaronpearce, first of all, thanks for the library! 

I'm currently in the process of setting it up with GRDB and I'm trying to understand how to conform my Models to HRecord. Especially around the inter play of archivedRecordData, record and zoneID. Do you have some examples of HRecord conformances that you can share?

Best
David

**Comment by aaronpearce** [comment, marked as answer]
👋 There's a small demo app available [here](https://github.com/aaronpearce/HarmonyDemoApp).

The usage of `zoneID` depends on your use case. You could use the default zone or create specific zone identifiers if you wish to share records in the future. Currently Harmony doesn't support sharing databases but I plan to add this in the future.
